src/biogeochem/CNFireFactoryMod.F90

```diff
@@ -17,6 +17,18 @@ module CNFireFactoryMod
   ! !PUBLIC ROUTINES:
   public :: CNFireReadNML         ! read the fire namelist
   public :: create_cnfire_method  ! create an object of class fire_method_type
+  public :: create_fates_fire_data_method  ! create an object of class fates_fire_base_type
```

Comment:
This is a FATES only module so should be renamed to have a FATES name. And it should likely be moved to be beside other FATES modules. Something like FATESFireDataFactory.F90 would make sense to me.

Reply:
updated per 08b1f8781a3dd3442e52e4719b840ae2559498b9